main.go

```diff
@@ -22,24 +22,41 @@ import (
 )
 
 var Options struct {
-	AssistedServiceScheme string `envconfig:"ASSISTED_SERVICE_SCHEME"`
-	AssistedServiceHost   string `envconfig:"ASSISTED_SERVICE_HOST"`
-	DataDir               string `envconfig:"DATA_DIR"`
-	HTTPSKeyFile          string `envconfig:"HTTPS_KEY_FILE"`
-	HTTPSCertFile         string `envconfig:"HTTPS_CERT_FILE"`
-	HTTPSCAFile           string `envconfig:"HTTPS_CA_FILE"`
-	ListenPort            string `envconfig:"LISTEN_PORT" default:"8080"`
-	HTTPListenPort        string `envconfig:"HTTP_LISTEN_PORT"`
-	MaxConcurrentRequests int64  `envconfig:"MAX_CONCURRENT_REQUESTS" default:"400"`
-	RHCOSVersions         string `envconfig:"RHCOS_VERSIONS"`
-	OSImages              string `envconfig:"OS_IMAGES"`
-	AllowedDomains        string `envconfig:"ALLOWED_DOMAINS"`
-	InsecureSkipVerify    bool   `envconfig:"INSECURE_SKIP_VERIFY" default:"false"`
-	ImageServiceBaseURL   string `envconfig:"IMAGE_SERVICE_BASE_URL"`
-	LogLevel              string `envconfig:"LOGLEVEL" default:"info"`
+	AssistedServiceScheme           string `envconfig:"ASSISTED_SERVICE_SCHEME"`
+	AssistedServiceHost             string `envconfig:"ASSISTED_SERVICE_HOST"`
+	DataDir                         string `envconfig:"DATA_DIR"`
+	HTTPSKeyFile                    string `envconfig:"HTTPS_KEY_FILE"`
+	HTTPSCertFile                   string `envconfig:"HTTPS_CERT_FILE"`
+
+	// Legacy/Depracted - Whenever possible please use ASSISTED_SERVICE_API_TRUSTED_CA_FILE
+	// This is a path to a CA file that can be used for TLS connections to the Assisted Service API
+	// this will be used for API calls back to the Assisted Service API
+	HTTPSCAFile                     string `envconfig:"HTTPS_CA_FILE"`
+
+	ListenPort                      string `envconfig:"LISTEN_PORT" default:"8080"`
+	HTTPListenPort                  string `envconfig:"HTTP_LISTEN_PORT"`
+	MaxConcurrentRequests           int64  `envconfig:"MAX_CONCURRENT_REQUESTS" default:"400"`
+	RHCOSVersions                   string `envconfig:"RHCOS_VERSIONS"`
+	OSImages                        string `envconfig:"OS_IMAGES"`
+	AllowedDomains                  string `envconfig:"ALLOWED_DOMAINS"`
+	InsecureSkipVerify              bool   `envconfig:"INSECURE_SKIP_VERIFY" default:"false"`
+	ImageServiceBaseURL             string `envconfig:"IMAGE_SERVICE_BASE_URL"`
+	LogLevel                        string `envconfig:"LOGLEVEL" default:"info"`
+
+	// This is a path to a CA file that will be used for fetching OS Images
+	// intended for scenarios where the customer is using an external service to host OS images.
```

Comment:
`user` preferred over `customer`

```suggestion
	// intended for scenarios where the OS images are served from a service that uses a custom CA
```